File: Leetcode/Easy/Problem-2574/Main.java
import java.util.*;

class Solution{
	public int[] leftRigthDifference(int[] nums){
		int sum=0;
		
		int[] leftSum=new int[nums.length];
		int[] rightSum=new int[nums.length];

		for(int i=0;i<nums.length;i++){
			leftSum[i]=sum;
			sum+=nums[i];
		}

		sum=0;

		for(int i=nums.length-1;i>=0;i--){
			rightSum[i]=sum;
			sum+=nums[i];
		}

		for(int i=0;i<nums.length;i++)
			nums[i]=Math.abs(leftSum[i]-rightSum[i]);

		return nums;
	}
}

class Main{
	public static void main(String[] args){
		Solution ob=new Solution();

		int[] nums={10,4,8,3};
		System.out.println(Arrays.toString(ob.leftRigthDifference(nums)));
	}
}

File: Leetcode/Easy/Problem-2574/Main.py
import math

class Solution:
	def leftRightDifference(self, nums: list[int]) -> list[int]:
		sumNums=0
		length=len(nums)

		leftSum=list()
		rightSum=list()

		for i in range(length):
			leftSum.append(sumNums)
			sumNums+=nums[i]

		sumNums=0

		for i in range(length-1,-1,-1):
			rightSum.append(sumNums)
			sumNums+=nums[i]

		for i in range(length):
			nums[i]=abs(leftSum[i]-rightSum[length-1-i])

		return nums

ob=Solution()

nums=[int(i) for i in input().split(',')]

print(ob.leftRightDifference(nums))
